### ISSUE TITLE
Errors while trying to use glowscript_filter

### ISSUE THREAD

**Issue by lameze** [BogdanKulinich/glowscript_filter#1]
Hi Bogdan, just to clarify about my comment of the errors I have noticed trying to use your plugin.

Firstly, I tried the zip file available here and got the following error on my JS console:
`
Uncaught SyntaxError: Unexpected number on view.php?id=5 (line 439) `

This error happens when I try to filter using one of the examples.

Then, I tried again, by cloning the repository, I got the following warning:

`Warning: file_get_contents(/home/simey/moodles/stable_master_review/moodle/filter/glowscript/init.js): failed to open stream: No such file or directory in /home/simey/moodles/stable_master_review/moodle/filter/glowscript/filter.php on line 7`

This happened eveywhere, even on the manage filters page.

To summarise, would be good some consistency between the two source, if you are going to work in another version for your plugin, create a different branch.

Thanks
